### DISCUSSION TITLE
Integrate Ant Media SDK With Deep AR SDK

### DISCUSSION THREAD

**Discussion by mariia-artamonova** [ant-media/Ant-Media-Server#5950] [FAQs] [answered]
Hi,

I'm new in JAVA and want to develop a live streaming app which required to have face beauty/makeup & background segmentation features. After some research, I decided to use Deep AR SDK.

From Deep AR's example with Agora [here](https://github.com/DeepARSDK/videocall-android-java), they are able to push external video frame by using Agora RTCEngine.pushExternalVideoFrame(frame). Are ant media server SDK able to do that? I also referred the ant media plugin [here](https://antmedia.io/plugins-will-make-ant-media-server-more-powerful/) but I has no idea on how to make it work with Deep AR SDK. May you guide me on how to integrate Ant Media SDK with Deep AR SDK? Or any way that are able to add face beauty/makeup & background segmentation features into Ant Media SDK in Android also can.

Thanks.

**Comment by yashtandon113** [comment, marked as answer]
@mariia-artamonova 
Thanks for your question.

Yes, the AMS is now compatible with the Deep AR Javascript SDK. Please check out the [Deep AR blogpost](https://antmedia.io/webrtc-video-streaming-deepar/) for more information.

You can also test the integration on [Ant Media Deep AR sample page](https://antmedia.io/webrtc-samples/deepar-publish-play/).

**Comment by D-Smarak** [comment]
Maybe it is working for Js, but i see no sign of progress in flutter sdk or IOS sdk for DeepAR integration as of today. Only if it would have worked, we could use this media server for our application.
